### DISCUSSION TITLE
“Today” Sensors / entities from Home Mini

### DISCUSSION THREAD

**Discussion by michaeljaddy** [BottlecapDave/HomeAssistant-OctopusEnergy#165] [Ideas]
Hi,

I have noticed in the Octopus app since getting a home mini that I am able to see a “today” page which shows the energy utilisation of the current day up until the present time - for both electricity and gas.
![F5956A5C-BAB9-4B70-9655-149046D380DD](https://user-images.githubusercontent.com/113230724/223695046-48683c65-73a7-4e99-bab5-e9812c1017e1.jpeg)

Is there any way of having these values as sensors / entities in HA?

Kind regards

Mike

**Comment by BottlecapDave** [comment]
As per the [readme](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy#home-mini), you can have Octopus Mini data come through into HA. Once these sensors are configured, you can then add them to the energy dashboard to get a daily view. You can also setup a utility meter sensor to track todays data.

**Comment by BottlecapDave** [comment]
I've also only just noticed that gas sensor data is now being pulled through, so I'm planning on adding this in a future release.

**Comment by cspiby** [comment]
How do we get exported power into the energy dashboard?

**Reply by BottlecapDave** [reply]
You could potentially create a template sensor which takes the previous consumption associated with your export and multiple by -1 to get a negative result? The home mini doesn't capture negative information to my knowledge.

**Reply by cspiby** [reply]
> You could potentially create a template sensor which takes the previous consumption associated with your export and multiple by -1 to get a negative result? The home mini doesn't capture negative information to my knowledge.

Perfect this is exactly what I've done and its working well now.
